src/components/App.tsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import "./App.scss";
 import { Header } from "./navigation/Header";
 import { Route, Routes } from "react-router-dom";
-import { Home } from "./Home/Home";
+import { Home } from "./home/Home";
```

Comment:
Can you rename the file back to `Home` instead of `home`? Because git didn't notice the change we had build issues with the import